src/store.js

```diff
@@ -0,0 +1,29 @@
+import {createStore, applyMiddleware, compose} from 'redux';
+import rootReducer from './reducers';
+import ReduxThunk from 'redux-thunk/';
+import {persistStore, persistReducer} from 'redux-persist';
+import storage from 'redux-persist/lib/storage';
+import autoMergeLevel1 from 'redux-persist/lib/stateReconciler/autoMergeLevel1';
+
+const persistConfig = {
+    key: 'root',
+    storage: storage,
+    // autoMergeLevel1 is the default anyway, but I want to be explicit
+    stateReconciler: autoMergeLevel1,
+    whitelist: ['auth', 'map']
```

Comment:
For this `persistConfig` configuration object that we're about to pass into `persistReducer`, here's what's going on:
 * `key: root` refers to that this is the root persistence that we're doing. We can do nested stuff, but for now let's keep it simple
 * `storage: storage` deals with *where* we'll be storing the offline data. The import we did above to grab this from `'redux-persist/lib/storage'` lets `redux-persist` choose the default location, which for React Native is `AsyncStorage`
 * `stateReconciler: autoMergeLevel1` specifies how `redux-persist`  should handle merge conflicts between dynamic state in the app and the state that's trying to be persisted (if there's a conflict while loading). `autoMergeLevel1` is the default, and in the docs it looks like the most reasonable approach, so I explicitly added it so we'd have visibility to which we're using, and can quickly swap out the value if we want to go a different way
 * `whitelist` refers to which parts of state we want to persist offline. This is really handy since it's asking for reducer's state keys from our `rootReducer`, so we can pass entire blocks of state. In our case we're now persisting the entire `auth` and `map` pieces of state

Reply:
Looking forward to seeing this in action!